src/popup/pages/AccountDetailsMultisigTokens.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="token-list-wrapper">
```

Comment:
I suggest to name a root class as a component name, but in kebab-case as every other component had.